code/game/turfs/open/floor/misc_floor.dm

```diff
@@ -9,6 +9,10 @@
 	icon_state = "rockvault"
 	floor_tile = /obj/item/stack/tile/plasteel
 
+/turf/open/floor/tile
+	icon_state = "floor_plates"
+	floor_tile = /obj/item/stack/tile/plasteel
```

Comment:
While you are at it, might want to add a proper stack item for this.
It's not necessary, but would be nice if people could replace floors using it, if they find a crowbar.